Azure/sonic-utilities
# [sonic-utilities]remove db_migrator logic for autoneg enable/disable

Signed-off-by: vaibhav-dahiya <vdahiya@microsoft.com>



#### What I did
This PR removes all the logic for db_migrator which went into #1817 , This is required because this will cause changing the logic to "0/1" to "on/off" in swss and cause a problem in upgrade. 
#### How I did it

#### How to verify it
autoneg enable/disable works with the implementation. Removing the db_migration logic so no check required.
#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
